_javascript/modules/components/toc/toc-desktop.js

```diff
@@ -0,0 +1,22 @@
+export class TocDesktop {
+  /* Tocbot options Ref: https://github.com/tscanlin/tocbot#usage */
+  static options = {
```

Comment:
"How to change TOC configuration?" is quite popular community question. The [solution](https://github.com/cotes2020/jekyll-theme-chirpy/discussions/1364#discussioncomment-10812644) I proposed will become more complex after this feature. I wonder if there is anything we can do to make it easier? Maybe you have an idea?

Reply:
There are ways to do this, such as reading tocbot's configuration from an external file, but this will run down performance and make the project bigger and bigger — [Feature Creep](https://en.wikipedia.org/wiki/Feature_creep).

I would suggest not to worry too much about this kind of customization, if they have a personal need, the easiest way is to change the configuration in the theme source code, and then compile a `post.min.js` and copy it to their Starter project.

Reply:
I was thinking more in the direction of extracting the options definition into a separate object that can be easily accessed and modified from within a custom script, for example. That way it will have no runtime impact while still being customizable.

Reply:
No matter how much we tweak it, we still have to maintain a piece of ugly "customization" code, don't we? I think the main goal of the project is to maintain a basic design, and since all the source code is open-source, "customization" should be the responsibility of users who are dissatisfied with parts of the original design.

Instead of exposing the config options for tobot, I think a more feasible approach would be to add a "expand/collapse" button for the TOC on the desktop view. But,  it is a topic outside the scope of this PR.